R/BASiCS_DetectHVG_LVG.R

```diff
@@ -154,99 +72,82 @@
 BASiCS_DetectVG <- function(
     Chain,
     Task = c("HVG", "LVG"),
-    PercentileThreshold = 0.1,
-    VarThreshold = 0.5,
-    ProbThreshold = 0.5,
+    PercentileThreshold = NULL, # 0.1,
+    VarThreshold = NULL, # 0.5,
+    ProbThreshold = 2/3, # 0.5,
     EFDR = 0.1,
-    OrderVariable = c("Prob", "GeneIndex","GeneName"),
+    OrderVariable = "Prob",
```

Comment:
The default value for this argument should be all valid values, so that match.arg correctly accepts any of them and rejects all others

Reply:
As is, OrderVariable can only be "Prob"

Reply:
(actually not the case, I was looking at the wrong matcharg)

Reply:
I tried that, but the unit tests complained about it. My manual fix was to use `match.arg(OrderVariable, c("Prob", "GeneIndex","GeneName")`

Reply:
It's helpful IMO to have all valid values for arguments like this displayed in the help page usage section "for free" rather than only in the argument documentation (which can become stale)

Reply:
Any idea what the complaint was? Seems to work locally for me

Reply:
I don't remember, I am afraid. I will re-run and let you know.

Reply:
It runs ok now, not sure what went wrong before 🤷‍♀️

Reply:
Life's great mysteries!

Reply:
done in cd5763f